src/constatnts.py

```diff
@@ -1,18 +1,50 @@
-POINT_SIDE_SIZE = 50  # [m]
-CELL_SIDE_SIZE = 10  # points per cell side size
-GRID_SIDE_SIZE = 10  # cells per grid side size
-WORLD_SIDE_SIZE = CELL_SIDE_SIZE * GRID_SIDE_SIZE
+import pandas as pd
+from data.generic import Range
+from data.measurment_data import Coordinates
+from data.simulation_run_parameters import CellSideCount, SimulationRunParameters
+
+POINT_SIDE_SIZE = 50  # [m] #TODO: right now that could be different from what we have from coordinates :v - need to be calculated
+POINTS_SIDE_COUNT = 100
```

Comment:
Does that mean we always assume square simulated space? Do we want it that way?

Reply:
i guess that was also done that way in previous version. If there will be time we could change that in a future